tests/test_uharfbuzz.py

```diff
@@ -283,6 +300,29 @@ def test_message_func_crash(self, blankfont):
         hb.shape(blankfont, buf)
 
 
+    def test_draw_funcs(self, opensans):
+        funcs = hb.DrawFuncs.create()
+        container = { "output_string": "" }
```

Comment:
I would just use a list here, and `container.append(...)` in the callbacks. Then you could do `output_string = "".join(container)`, although for the sake of the test I think the fact that this is SVG is not relevant.